frontend/webpack.config.ts

```diff
@@ -34,6 +35,11 @@ const virtualModules = new VirtualModulesPlugin();
 const overpassTest = /overpass-.*\.(woff2?|ttf|eot|otf)(\?.*$|$)/;
 const sharedVendorTest = new RegExp(`node_modules\\/(${sharedVendorModules.join('|')})\\/`);
 
+const sharedPatternFlyCoreModules = ['@patternfly/react-core', '@patternfly/react-table'];
```

Comment:
This could be `sharedVendorModules.filter((m) => m.startsWith('@patternfly/'))` or, if we need a smaller subset, let's declare it in `shared-modules.ts` as a variable next to `sharedVendorModules` to have this information in one place.

Reply:
We don't want all of the PatternFly packages, just these two.